src/components/button/button.jsx

```diff
@@ -0,0 +1,33 @@
+// Copied from scratch-gui for now
+
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import styles from './button.css';
+
+const ButtonComponent = ({
+    className,
+    onClick,
+    children,
+    ...props
+}) => (
+    <span
+        className={classNames(
+            styles.button,
+            className
+        )}
+        role="button"
+        onClick={onClick}
+        {...props}
```

Comment:
It looks like {...props} would be empty?

Reply:
It is for current instances – the idea was to pass props to the button itself that are custom for a given implementation, should it arise.

I can take this out, though, if we don't want it for now!

Reply:
Oh cool, I didn't know that was a thing you could do